scripts/generator/graphql_generator/csharp/SDK/Android/AndroidNativeCheckout.cs.erb

```diff
@@ -23,7 +24,7 @@ namespace <%= namespace %>.SDK.Android {
             CartState = cartState;
         }
 
-        public bool CanShowPaymentSetup() {
+        public bool CanShowPaymentSetup(PaymentSettings paymentSettings) {
             // TODO
             return true;
```

Comment:
nope this is for opening a wallet app to set up Apple Pay/Android Pay. [`openPaymentSetup`](https://developer.apple.com/documentation/passkit/pkpasslibrary/1617100-openpaymentsetup)